src/Symfony/Bundle/FrameworkBundle/Resources/public/css/body.css

```diff
@@ -39,7 +39,7 @@ build: 56
     font-family: Georgia, "Times New Roman", Times, serif;
     font-size: 20px;
     color: #313131;
-    word-break: break-all;
+    word-break: break-word;
```

Comment:
Should be `word-wrap: break-word`: `break-word` is an unofficial value for `word-break`.